zndraw/server/events.py

```diff
@@ -576,9 +576,13 @@ def modifier_run(data: dict):
         "modifier:run:enqueue", to=f"webclients_{session['token']}", include_self=False
     )
     # split into separate streams based on the modifier name
-    tasks.run_modifier(
+    queue_name = tasks.run_modifier(
         f"http://127.0.0.1:{current_app.config['PORT']}", session["token"], data
     )
+    if queue_name == "slow":
+        queue_positions = get_queue_position(queue_name)
```

Comment:
do we allow read-only database access in events? This might potentially be blocking.